docs/api/scripts_unified_code_assistant.md

```
@@ -0,0 +1,145 @@
+*No description available.*
+
+#### `_extract_lint_issues`
+*No description available.*
+
+#### `_extract_complexity_issues`
+*No description available.*
+
+
+## `scripts\unified_code_assistant\module_summarizer`
+
+**🧠 Docstring Summary**
```

markdownlint-cli2 (0.17.2)

Emphasis used instead of a heading
null

(MD036, no-emphasis-as-heading)